sdk/eventhub/azure-eventhub/azure/eventhub/_client_base.py

```diff
@@ -154,6 +156,7 @@ def _parse_conn_str(
         entity,
         str(shared_access_signature) if shared_access_signature else None,
         shared_access_signature_expiry,
+        bool(use_development_emulator) if use_development_emulator else None,
```

Comment:
Is this a string True/False value? b/c I think it shows up as true if the string is not empty even if it's "false"